addons/binding/org.openhab.binding.nest/src/main/java/org/openhab/binding/nest/handler/BaseNestHandler.java

```diff
@@ -0,0 +1,14 @@
+package org.openhab.binding.nest.handler;
+
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+abstract class BaseNestHandler extends BaseThingHandler {
```

Comment:
Please add a JavaDoc with `@author` tag containing a real name of the author. See [Code Style](http://docs.openhab.org/developers/development/guidelines.html#a-code-style).